src/songs.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import requestPromise from 'request-promise'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemAvatar from '@material-ui/core/ListItemAvatar'
+import Avatar from '@material-ui/core/Avatar'
+import ListItemText from '@material-ui/core/ListItemText'
+import { withStyles } from '@material-ui/core/styles'
+
+const OpaqueList = withStyles({
+  root: {
+    backgroundColor: 'rgba(255, 255, 255, 0.8)',
+    maxWidth: '600px',
+    margin: 'auto'
+  }
+})(List)
+
+const NarrowListItem = withStyles({
+  root: {
+    width: '80%'
+  }
+})(ListItem)
+
+const SongText = withStyles({
+  primary: {
+    lineHeight: '1rem',
+    fontSize: '0.8rem'
+  },
+  secondary: {
+    fontSize: '0.7rem'
+  }
+})(ListItemText)
+
+export default class Songs extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      songs: []
+    }
+  }
+
+  componentDidMount() {
+    const songDataRequest = {
+      url: 'https://api.spotify.com/v1/playlists/' +
```

Comment:
It would be good to declare the base URL for spotify in a `constants.js` file or something.

```js
export const SPOTIFY_API = 'https://api.spotify.com/v1/'
```